### DISCUSSION TITLE
Will foxglove-ws provide the Golang version？

### DISCUSSION THREAD

**Discussion by StarCsu** [foxglove/community#791] [Q&A] [answered]
Recently i will use foxglove-ws,but out backend server's mainly language is golang that foxglove-ws lacks golang version;

Will you privide the Golang module?

If you have no plans in the short term, can I provide the Golang version of the code？

**Comment by defunctzombie** [comment, marked as answer]
We have no plans to provide a GoLang base server but would be happy to review a PR to https://github.com/foxglove/ws-protocol adding one or link to community versions if you'd like to maintain it in your own repo.

**Reply by StarCsu** [reply]
OK，i got it
